hibernate-core/src/test/java17/org/hibernate/orm/test/records/MergeRecordPropetyTestCase.java

```diff
@@ -0,0 +1,75 @@
+package org.hibernate.orm.test.records;
+
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.JiraKey;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+import jakarta.persistence.Embeddable;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DomainModel(annotatedClasses = {
+		MergeRecordPropetyTestCase.MyEntity.class
+})
+@SessionFactory
+@JiraKey("HHH-16759")
+public class MergeRecordPropetyTestCase {
```

Comment:
"MergeRecordPropetyTestCase" should be "MergeRecordPropertyTestCase" (it's one of those things that either you see it at once, or you will never see it - our brain plays this kind of tricks on us ;) )

Reply:
I see ... time to check my eyesight :-(